response_operations_ui/views/collection_exercise.py

```diff
@@ -38,9 +38,14 @@ def view_collection_exercise(short_name, period, error=None, ci_loaded=False, ex
     ]
 
     ce_state = ce_details['collection_exercise']['state']
-    show_set_live_button = ce_state == 'READY_FOR_REVIEW'
+    show_set_live_button = ce_state in ('READY_FOR_REVIEW', 'FAILEDVALIDATION')
     locked = ce_state in ('LIVE', 'READY_FOR_LIVE', 'EXECUTION_STARTED', 'VALIDATED', 'EXECUTED')
     processing = ce_state in ('EXECUTION_STARTED', 'EXECUTED', 'VALIDATED')
+    validation_failed = ce_state == 'FAILEDVALIDATION'
```

Comment:
🎨 Super minor, but maybe `ce_state is 'FAILEDVALIDATION'` just so there is less = all over the place ^^